dashboard/src/scenes/report/view.jsx

```diff
@@ -188,12 +186,17 @@ const itemsForReportsSelector = selectorFamily({
         observations[observation._id] = observation;
       }
 
+      //      const { startDate, endDate } = defaultIsoDates;
+      const startDateForReports = dayjsInstance(defaultIsoDates.isoStartDate).format("YYYY-MM-DD");
+      const endDateForReports = dayjsInstance(defaultIsoDates.isoEndDate).format("YYYY-MM-DD");
+      console.log("startDateForReports", startDateForReports);
+      console.log("endDateForReports", endDateForReports);
       for (const report of allReports) {
         if (!filterItemByTeam(report, "team")) continue;
         const date = report.date;
-        const { isoStartDate, isoEndDate } = defaultIsoDates;
-        if (date < isoStartDate) continue;
-        if (date >= isoEndDate) continue;
+
+        if (date < startDateForReports) continue;
+        if (date > endDateForReports) continue;
```

Comment:
Et on exclut seulement si la date de fin est strictement supérieure, parce que comme elle a été mise à "end of the day" elle tombe le même jour.